src/sentry/static/sentry/app/views/eventsV2/eventDetails/content.tsx

```diff
@@ -155,9 +167,20 @@ class EventDetailsContent extends AsyncComponent<Props, State> {
           />
           <EventHeader event={event} />
           <HeaderBottomControls>
-            <StyledButton size="small" onClick={this.toggleSidebar}>
-              {isSidebarVisible ? 'Hide Details' : 'Show Details'}
-            </StyledButton>
+            <ButtonBar gap={1}>
+              <StyledButton size="small" onClick={this.toggleSidebar}>
+                {isSidebarVisible ? t('Hide Details') : t('Show Details')}
+              </StyledButton>
+              {transactionSummaryTarget && (
+                <StyledButton
+                  priority="primary"
+                  size="small"
+                  to={transactionSummaryTarget}
+                >
+                  {t('Go to summary')}
```

Comment:
Should this be in title case to be consistent with the details button?

Reply:
Good idea.